jest.config.js

```diff
@@ -15,6 +15,7 @@ module.exports = {
     collectCoverageFrom: ['app/**/*.{js,jsx,ts,tsx}'],
     coverageReporters: ['lcov', 'text-summary'],
     testPathIgnorePatterns: ['/node_modules/'],
+    coveragePathIgnorePatterns: ['/node_modules/', '/components/', '/screens/'],
```

Comment:
Maybe there is some webpack magic going on, but shouldn't this be...
```suggestion
    coveragePathIgnorePatterns: ['/node_modules/', '/app/components/', '/app/screens/'],
```

Reply:
I did confirm it works before submitting. I think it's because

```
collectCoverageFrom: ['app/**/*.{js,jsx,ts,tsx}']
```

and it's ignoring patterns, not paths